ergo/distributions/histogram.py

```diff
@@ -104,8 +104,56 @@ def sample(self):
     def rv(self):
         raise NotImplementedError
 
-    def normalize(self):
-        return HistogramDist(self.logps, 0, 1)
+    # type ignore here because I got ahead of the rest of the repo
+    # with refactoring to use a Scale rather than a scale_min/scale_max
+    def normalize(self, true_scale: scale.Scale = None):  # type: ignore
+        """
+        Normalize the histogram onto [0,1]
+
+        Setting a true_scale allows you to express that the histogram to be normalized
+        did not cover the entire scale of interest.
+
+        E.g. -- imagine my histogram only has probability mass from 2 to 5,
+        but I'm interested in p over [0,10].
+        So I'll use Scale(0,10), and I'll get back a histogram with additional
+        bins with 0 probability to cover the area from 0 to 2 and 5 to 10,
+        where my histogram doesn't place any probability.
+
+        :param true_scale: the full scale that I'm interested in probability over.
+        """
+        # if true_scale is not provided, assume that the histogram has
+        # entries over the entire scale of interest
+        if not true_scale:
```

Comment:
interesting idea --I don't think we'd want it to be a truthiness value for the scale, because the scale can be missing a `scale_min` or `scale_max` to indicate that that end of the scale is open/goes to +/-infinity

but maybe it could be a property like `scale.closed`

Reply:
ohhhh got it

I was misinterpreting your comment, and actually my code is wrong/bad (basically I was accidentally applying a JS idiom to Python -- I know better but forgot as I was writing this)

thanks for clarifying

should be `if true_scale is None`